### PR TITLE
Execute lambda with qualifier if lambda function version is specified

### PR DESCRIPTION
**Pull request by vivianho**
Lyft uses alias versioning for its lambda, so I am adding it here as an optional config variable.

### CODE REVIEW

**Comment by vivianho** [comment]
I'm not sure how vendoring in Go works. Let me know if I should be running some command or something instead!

**Comment by vivianho** [comment]
@edulop91

**Comment by edulop91** [comment]
for vendor we're currently using [dep](https://github.com/golang/dep). you can usually resolve/add new dependencies with `dep ensure`. and upgrade versions with `dep ensure -update`

File: pkg/bless/client.go [hunk 1]
```diff
@@ -198,7 +198,12 @@ func (c *Client) getCert(ctx context.Context, payload *LambdaPayload) (*LambdaRe
 	if err != nil {
 		return nil, errors.Wrap(err, "Could not serialize lambda payload")
 	}
-	responseBytes, err := c.Aws.Lambda.Execute(ctx, c.conf.LambdaConfig.FunctionName, payloadB)
+	var responseBytes []byte
+	if c.conf.LambdaConfig.FunctionVersion != nil {
+		responseBytes, err = c.Aws.Lambda.ExecuteWithQualifier(ctx, c.conf.LambdaConfig.FunctionName, *c.conf.LambdaConfig.FunctionVersion, payloadB)
```
Comment (edulop91) [inline pkg/bless/client.go]: I think aws.Lambda already deals with optional qualifiers, what do you think about making that an optional param into ExecuteWithQualifier and letting AWS deal with that conditional? Would involve merging https://github.com/chanzuckerberg/go-misc/pull/22

Comment (edulop91) [reply pkg/bless/client.go]: That way we get rid of this conditional

File: pkg/config/config.go [hunk 2]
```diff
@@ -84,6 +84,8 @@ type LambdaConfig struct {
 	RoleARN *string `yaml:"role_arn,omitempty"`
 	// Bless lambda function name
 	FunctionName string `yaml:"function_name"`
+	// Bless lambda function version (lambda alias or version qualifier)
+	FunctionVersion *string `yaml:"function_version,omitempty"`
```
Comment (edulop91) [inline pkg/config/config.go]: :+1:

File: pkg/bless/client.go (hunk 1 above)
Comment (vivianho) [reply pkg/bless/client.go]: Yeah! That's what I was trying to achieve but I'm just being noob at go :) I will wait until you merge that, and then re-run `dep ensure -update`

Comment (edulop91) [reply pkg/bless/client.go]: done!

Comment (edulop91) [reply pkg/bless/client.go]: doing pkg updates https://github.com/chanzuckerberg/blessclient/pull/81 because somehow i lost the dep version lock file

**Comment by edulop91** [comment]
https://travis-ci.com/chanzuckerberg/blessclient/jobs/158945877 just curious, are you able to see the travis jobs?

**Comment by vivianho** [comment]
Ah yeah I am, sorry I didn't realize I made a mistake! Fixed.

**Comment by edulop91** [comment]
thanks!

**Pull request merged**